store/mockstore/unistore/tikv/dbreader/db_reader.go

```diff
@@ -289,6 +318,24 @@ func (r *DBReader) ReverseScan(startKey, endKey []byte, limit int, startTS uint6
 	return nil
 }
 
+// CheckWriteItemForRcRead checks the data version if `RcCheckTS` isolation level is used.
+func (r *DBReader) CheckWriteItemForRcRead(readTS uint64, item *badger.Item) error {
```

Comment:
```suggestion
func (r *DBReader) CheckWriteItemForRcCheckTSRead(readTS uint64, item *badger.Item) error {
```

Consider renaming it with "RcCheckTS".